.publicApi/Microsoft.ApplicationInsights.AspNetCore.dll/net451/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,4 @@
+Microsoft.ApplicationInsights.AspNetCore.Extensions.ApplicationInsightsServiceOptions.EnableDiagnosticsTelemetryModule.get -> bool
+Microsoft.ApplicationInsights.AspNetCore.Extensions.ApplicationInsightsServiceOptions.EnableDiagnosticsTelemetryModule.set -> void
+Microsoft.ApplicationInsights.AspNetCore.Extensions.ApplicationInsightsServiceOptions.EnableTelemetryConfigurationActiveBackwardsCompatibility.get -> bool
```

Comment:
Consider naming this to something like `ConfigureTelemetryConfigurationActive`. There is no need of 'backwardcompatibility' in field name in my opinion.

Reply:
I would prefer if we don't use "Configure" twice.
How do you feel about "EnableTelemetryConfigurationActive" or "EnableLegacyConfiguration"?

Reply:
Not sure if "legacy" is correct. TelemetryConfiguration itself is not legacy! only the usage of .Active is.

EnableTelemetryConfigurationActive - this is misleading. Even with this changes, TC.Active is still available. Its just that it wont be configured correctly.

How does `EnableActiveTelemetryConfigurationSetup` sound?